dev/ColorPicker/APITests/ColorPickerTests.cs

```diff
@@ -232,6 +246,37 @@ public void ValidateHueRange()
             IdleSynchronizer.Wait();
         }
 
+        [TestMethod]
+        public void ValidateFractionalWidthDoesNotCrash()
+        {
+            ManualResetEvent spectrumLoadedEvent = new ManualResetEvent(false);
+
+            RunOnUIThread.Execute(() =>
+            {
+                ColorSpectrum colorSpectrum = new ColorSpectrum();
+                colorSpectrum.Width = 300.75;
+                colorSpectrum.Height = 300.75;
+                colorSpectrum.Loaded += (sender, args) =>
+                {
+                    var spectrumRectangle = VisualTreeUtils.FindVisualChildByName(colorSpectrum, "SpectrumRectangle") as Rectangle;
+
+                    if (spectrumRectangle != null)
```

Comment:
Should this ever be null? If it is null, the test will timeout. It would probably better for it to null deref, since that would be clearer.

Reply:
Good point.  I'll remove the null check.

Reply:
I can also add an explicit Verify.IsNotNull call to make it explicit that this should never be null, which should make error logging even more clear.